### PR TITLE
replaced `me` with `self`

### PR DESCRIPTION
**Pull request by aviRon012**
replaced `me` with `self` across documentation.

### CODE REVIEW

File: proposals/p0722.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
             -   [Full receiver type](#full-receiver-type)
             -   [Method names line up](#method-names-line-up)
             -   [Receiver in square brackets](#receiver-in-square-brackets)
-            -   [Receiver parameter is named `me`](#receiver-parameter-is-named-me)
+            -   [Receiver parameter is named `self`](#receiver-parameter-is-named-self)
```
Comment (aviRon012) [inline proposals/p0722.md:25]: Maybe this should remain unchanged to reflect the history of the project?

File: proposals/p0722.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ the call and the function declaration. That motivated putting the receiver
 pattern inside square brackets (`[`...`]`) with deduced parameters. There were
 concerns, however, that this parameter did not act like deduced parameters.
 
-##### Receiver parameter is named `me`
+##### Receiver parameter is named `self`
```
Comment (aviRon012) [inline proposals/p0722.md:168]: Same question.

File: proposals/p0722.md [hunk 3]
```diff
@@ -176,7 +176,7 @@ associated with the type, like
 
 Finally, we wanted the bodies of methods to access members of the object using
 an explicit member access. This means we need the name used to access members to
-be short, and so we chose the name `me`.
+be short, and so we chose the name `self`.
```
Comment (aviRon012) [inline proposals/p0722.md:179]: Same question.

File: proposals/p0722.md (hunk 1 above)
Comment (chandlerc) [reply proposals/p0722.md:25]: All of the proposals should remain unchanged.

We generally only do critical typo fixes (misspelled someone's name or something) or updating of links in proposals. While it's fine to try to keep *navigating* the proposal working (link updates), we shouldn't try to make them living documents. It's OK (and even good) for them to reflect the state of the language and project when they were accepted.

**Review by chandlerc — CHANGES_REQUESTED** [review]
(see inline reply, sorry I forgot to attach it to my review and posted it on its own)

**Comment by aviRon012** [comment]
since most changes were undone I restarted from scratch and created PR 2631

**Pull request closed without merging**